.github/workflows/sylius-frankenphp.yml

```
@@ -0,0 +1,241 @@
+name: FrankenPHP Docker Image CI for Sylius
+
+on:
+    workflow_dispatch: ~
+    push:
+        branches: [ main, frankenphp ]
+        paths:
+            - 'frankenphp/**'
+            - 'WORKSPACE'
+    schedule:
+        - cron: '0 20 * * 5'
+
+concurrency:
+    group: sylius_frankenphp_build
+    cancel-in-progress: false
+
+jobs:
+
+    build:
+
+        name: "Sylius FrankenPHP - ${{ matrix.frankenphp }}${{ matrix.php }} ${{ matrix.distro }}"
+
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                frankenphp: [ "1.5-php" ]
+                php: [ "8.2", "8.3", "8.4" ]
+                distro: [ "alpine" ]
+
+        steps:
+
+            - uses: actions/checkout@v4
```

Is  this top level whiteline necessary?

```suggestion
        steps:
            - uses: actions/checkout@v4
```